modules/_node-scripts/cssLoad/writeCSSLoaderJavaScriptModules.mjs

```diff
@@ -0,0 +1,37 @@
+import fs from 'fs/promises';
+import path from 'path';
+
+import getFlatName from '../util/getFlatName.mjs';
+
+export default async function writeCSSLoaderJavaScriptModules(projectExports, projectWebContextPath) {
+	if (!projectExports) {
+		return;
+	}
+
+	return Promise.all( 
+		projectExports
+			.filter(moduleName => moduleName.endsWith('.css'))
+			.map(moduleName => writeCSSLoaderJavaScriptModule(projectWebContextPath, moduleName))
+	);
+}
+
+async function writeCSSLoaderJavaScriptModule(webContextPath, moduleName) {
```

Comment:
same as above, explicit naming for "exports" since we wouldn't use this for something not under the `__liferay__/exports` path

Reply:
done